app/services/FinancialInstitutionUpdateService.scala

```diff
@@ -25,7 +26,9 @@ import pages.addFinancialInstitution.IsRegisteredBusiness.{IsTheAddressCorrectPa
 import pages.addFinancialInstitution._
 import pages.changeFinancialInstitution.ChangeFiDetailsInProgressId
 import play.api.libs.json.{Json, Reads}
+import play.api.mvc.Results.Redirect
 import repositories.SessionRepository
+import uk.gov.hmrc.http.HeaderCarrier
```

Comment:
There are many unused imports - can we please clean this up, and also check any other files. It will show on compilation warning which unused imports are present